test/jdk/ProblemList.txt

```diff
@@ -529,6 +529,7 @@ java/awt/Graphics2D/DrawString/DrawRotatedStringUsingRotatedFont.java 8266283 ge
 java/awt/KeyboardFocusmanager/TypeAhead/ButtonActionKeyTest/ButtonActionKeyTest.java 8257529 windows-x64
 
 java/awt/Window/GetScreenLocation/GetScreenLocationTest.java 8225787 linux-x64
+java/awt/Dialog/MakeWindowAlwaysOnTop/MakeWindowAlwaysOnTop.java 8252813 macosx-aarch64
```

Comment:
Looks like this bug is a closed one, can it be open?

Reply:
Maybe we should use JDK-8266243 instead. Other one has other tests as well.